tests/cypress/support/selectors.js

```diff
@@ -10,4 +10,10 @@ export default {
     exportViewPopover: "[data-test='export-view-popover']",
     logoImage: "[data-test='logo-image']",
     loader: "[data-test='loader']",
+    videoGuide: "[data-test=video-guide-button]",
+    timelineButton: "app-timeline > .ion-no-padding",
+    list : "//*[starts-with(@class,'active')]/parent::ion-item/ion-button[starts-with(@class,'active')]",
```

Comment:
Please avoid using general selectors as they may change during development.

Add a test specific tag attribute like `[data-test='lead-time-buttons']` as in the other selector tags. [Here is the Cypress documentation for reference.](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)

The key `list` is not descriptive for any reader to understand. Please be more specific in variable names. Like `leadTimeButttonList` or `leadTimeButtons` or `leadTimeButtonContainer`

Reply:
done changes.

i have created a pull request again now.